### PR TITLE
leaflet-radar

### PR DESCRIPTION
**Pull request by cygis2011**

### CODE REVIEW

File: docs/_plugins/overlay-animations/leaflet-radar.md
Comment (Falke-Design) [inline docs/_plugins/overlay-animations/leaflet-radar.md:8]:
```suggestion
compatible-v0: false
```

File: docs/_plugins/overlay-animations/leaflet-radar.md
Comment (Falke-Design) [inline docs/_plugins/overlay-animations/leaflet-radar.md:10]:
```suggestion
---

Radar sector scan animation
```

**Review by Falke-Design — CHANGES_REQUESTED** [review]

**Review by Falke-Design — APPROVED** [review]

**Comment by Falke-Design** [comment]
We need to wait until another Maintainer approved it

**Review by mourner — APPROVED** [review]

**Pull request merged**
